src/Components/Web.JS/package.json

```diff
@@ -5,9 +5,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "preclean": "yarn install --mutex network",
+    "clean": "node node_modules/rimraf/bin.js ./dist",
+    "prebuild": "yarn run clean && yarn install --mutex network",
     "build": "yarn run build:debug && yarn run build:production",
-    "build:debug": "cd src && webpack --mode development --config ./webpack.config.js",
-    "build:production": "cd src && webpack --mode production --config ./webpack.config.js",
+    "build:debug": "cd src && node ../node_modules/webpack-cli/bin/cli.js --mode development --config ./webpack.config.js",
```

Comment:
Is this the workaround for not requiring webpack globally? Worth a comment (haha it's json you CANT)

Reply:
This is what signalr does. Requiring webpack installed globally isn't a good pattern it turns out.